src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.netcoreapp.cs

```diff
@@ -2,18 +2,34 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
+using System.Runtime.InteropServices;
 using System.Security.AccessControl;
+using Microsoft.Win32.SafeHandles;
 
 namespace System.IO
 {
     public static partial class FileSystemAclExtensions
     {
+        #region Private members
+
+        private const int GENERIC_READ = unchecked((int)0x80000000);
```

Comment:
Should this better be under Common/Interop ? We have this constant duplicated 5+ times, adding more is not not helping.

Reply:
Done. I also fixed the other ones we were consuming within System.IO.* projects.